tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -30,19 +30,14 @@ spec:
       description: The workspace where the snapshot spec and data json files reside
   steps:
     - name: push-snapshot
-      image: quay.io/konflux-ci/release-service-utils:26f77b0933b19e7318bacf5dee297a6c113bb57e
+      image: quay.io/konflux-ci/release-service-utils:12841cd2a35711b24a6749a93569d4118b173dcb
       script: |
         #!/usr/bin/env bash
         set -eux
 
-        push_image () { # Expected arguments are [origin_digest, name, containerImage, repository, tag, arch]
+        push_image () { # Expected arguments are [origin_digest, name, containerImage, repository, tag, arch, oras_args]
           # note: Inspection might fail on empty repos, hence `|| true`
-          destination_digest=$(
-            skopeo inspect \
-            --override-arch "$6" \
-            --no-tags \
-            --format '{{.Digest}}' \
-            "docker://$4:$5" 2>/dev/null || true)
+          destination_digest=$(oras resolve --registry-config "$AUTH_FILE" ${7:-} "$4:$5" 2>/dev/null || true)
```

Comment:
it is needed, otherwise if you leave as `$7` or `${7}`, you can see this:

```
[run-task : push-snapshot] + push_image sha256:0b770b6ec5414e841167266c96af42404e9f7049f72c21d0ab312e07c9403197 comp registry.io/image:tag registry.io/image tag amd64
[run-task : push-snapshot] /tekton/scripts/script-0-gch5g: line 76: $7: unbound variable

```

Reply:
I guess that happens if `oras_args` remains as `()`? Because every call to `push_image` passes `oras_args[@]`

Reply:
seems so

Reply:
When you rebase can you add a comment saying like `# We need this default value for when oras_args is equal to ()`?